bot-components/GitHub_queries.mli

```diff
@@ -0,0 +1,104 @@
+val send_graphql_query :
+     bot_info:Utils.bot_info
+  -> < parse: Yojson.Basic.t -> 'a
+     ; query: string
+     ; variables: Yojson.Basic.t
+     ; .. >
+  -> ('a, string) result Lwt.t
+
+type backport_info =
+  {backport_to: string; request_inclusion_column: int; backported_column: int}
+
+type full_backport_info =
+  {backport_info: backport_info list; rejected_milestone: string}
+
+val get_backport_info :
+  bot_info:Utils.bot_info -> string -> full_backport_info option
```

Comment:
This function doesn't return a Lwt.t because it doesn't make any calls to GraphQL or Rest queries. So I moved it to the Bot_components.Utils module since it's not a query: it parses a string containing some backport information.